recipes/r-diptest/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = '0.75-7' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-diptest
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: diptest_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/diptest_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/diptest/diptest_{{ version }}.tar.gz
+  sha256: 462900100ca598ef21dbe566bf1ab2ce7c49cdeab6b7a600a50489b05f61b61b
+
+
+build:
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - {{native}}gcc-libs   # [win]
+    - libgcc               # [not win]
+
+test:
+  commands:
+    - $R -e "library('diptest')"  # [not win]
+    - "\"%R%\" -e \"library('diptest')\""  # [win]
+
+
+about:
+  home: https://CRAN.R-project.org/package=diptest
+  license: GPL (>= 2)
```

Comment:
It is better to use the SPDX short version `GPL-2.0`, but I am not sure how to represent the `>=` in that case.